pkg/app/AppCrudOperationService.go

```diff
@@ -289,6 +307,80 @@ func (impl AppCrudOperationServiceImpl) GetAppMetaInfo(appId int) (*bean.AppMeta
 	}
 	return info, nil
 }
+
+func (impl AppCrudOperationServiceImpl) GetHelmAppMetaInfo(appId string) (*bean.AppMetaInfoDto, error) {
+
+	// adding separate function for helm apps because for CLI helm apps, apps can be of form "1|clusterName|releaseName"
+	// In this case app details can be fetched using app name / release Name.
+
+	appIdSplitted := strings.Split(appId, "|")
+
+	app := &app.App{}
+	var err error
+
+	if len(appIdSplitted) > 1 {
+		appName := appIdSplitted[2]
```

Comment:
if len(appIdSplitted) > 1 then how can we be sure that index 2 exists? we can only say that index 1 exists for sure

Reply:
In this case appId will be of form 1|clusterName|releaseName.